packages/client/src/raffles/getRaffle.ts

```diff
@@ -0,0 +1,22 @@
+import { adaptError } from '../helpers/client/formatError';
+import client from '../helpers/client';
+import join from 'proper-url-join';
+import type { GetRaffle } from './types/getRaffle.types';
+
+/**
+ * Method responsible for getting a raffle by its id.
+ *
+ * @param raffleId - Raffle identifier.
+ * @param config - Custom configurations to send to the client instance (axios).
+ *
+ * @returns Promise that will be resolved when the call to the endpoint finishes.
+ */
+const getRaffle: GetRaffle = (raffleId, config) =>
+  client
```

Comment:
@boliveira I know this is against the other client's at blackout, but any reason for us to not go with the `try{ await... } catch(error) {throw(adaptError); } ` approach?

Reply:
In terms of behavior, there is no reason. This is a style matter and I am ok with both ways but as we have been using it throughout the client package, I think we should keep it.

Reply:
Hum oki, personally i prefer the other way, visually its easier to see whats going on, but its a personal matter I guess :D